### PR TITLE
fix: text-shadow

### PR DESCRIPTION
**Pull request by LuciNyan**
### Description
- Fix text-shadow.
- Refactor the **text** to improve readability. Since the refactoring changes may be extensive, to prevent bugs caused by refactoring from being hard to locate, it will be implemented in multiple PRs step-by-step.

### CODE REVIEW

**Comment by Jackie1210** [comment]
Great! Could you add a test on the issue of `text-shadow`?

**Comment by LuciNyan** [comment]
> Great! Could you add a test on the issue of `text-shadow`?

Sure! Thanks for your suggestion!

File: src/handler/expand.ts [hunk 1]
```diff
@@ -172,21 +173,22 @@ function handleSpecialCase(
...
-        const styles = getStylesForProperty('textShadow', shadow, true)
-        for (const k in styles) {
-          if (!result[k]) {
-            result[k] = [styles[k]]
-          } else {
-            result[k].push(styles[k])
-          }
+    const result = {}
+
+    const shadows = value.includes(',') ? splitEffects(value) : [value]
```
Comment (Jackie1210) [inline src/handler/expand.ts:178]: I think we can just use `splitEffects(value)`? @LuciNyan

File: src/text/index.ts [hunk 2]
```diff
@@ -459,13 +439,13 @@ export default async function* buildTextNodes(
...
   if (parentStyle.textShadowOffset) {
-    let { textShadowColor, textShadowOffset, textShadowRadius } =
-      parentStyle as any
-    if (!Array.isArray(parentStyle.textShadowOffset)) {
-      textShadowColor = [textShadowColor]
-      textShadowOffset = [textShadowOffset]
-      textShadowRadius = [textShadowRadius]
-    }
+    const { textShadowColor, textShadowOffset, textShadowRadius } = parentStyle
+    console.log(
```
Comment (Jackie1210) [inline src/text/index.ts:443]: should remove

Comment (LuciNyan) [reply src/text/index.ts]: Done!

File: src/handler/expand.ts (hunk 1 above)
Comment (LuciNyan) [reply src/handler/expand.ts]: Great! and 国庆快乐！🎉

**Review by Jackie1210 — APPROVED** [review]

**Review by shuding — APPROVED** [review]
Thank you!!

**Pull request merged**